src/python/turicreate/toolkits/object_detector/_tf_model_architecture.py

```diff
@@ -262,8 +262,10 @@ def tiny_yolo(self, inputs, output_size=125):
                     strides = [1, 2, 2, 1]
                 else:
                     strides = [1, 1, 1, 1]
-
-                net = self.pooling_layer(net, pool_size=[1, 2, 2, 1], strides=strides, name='pool%d_' % idx)
+                if idx < 6 :
```

Comment:
Can we only use one `if else`? Set both `stride` and `padding` in the same `if else`.

Reply:
Yeah makes sense.